Makefile.in

```diff
@@ -549,6 +549,11 @@ copy.libghdl.deps.so:
 
 uninstall.libghdl:
 	$(RM) $(DESTDIR)$(libdir)/$(libghdl_name)
+	$(RM) $(DESTDIR)$(libdir)/libghdl.a
+	$(RM) $(DESTDIR)$(libdir)/libghdl.link
+	$(RM) $(DESTDIR)$(libdir)/libghw.dll
```

Comment:
What about the other shared library extensions `*.so` and `*.dylib`. Are these handled at other positions or should this rule use a variable? I believe there was something like `${SOEXT}`.

Reply:
thank you for your help!
I think I found it and it should now take care of .so and .dylib